src/component/FetchItems.js

```diff
@@ -111,8 +112,51 @@ const FetchItems = ({ token, setToken, firestore }) => {
                 }
               });
 
+              // Update state whenever filter input changes
+              const handleFilterChange = event => {
+                setFilteredInput(event.target.value.toLowerCase());
+              };
+
+              const handleClearFieldButtonClick = event => {
+                event.preventDefault();
+                setFilteredInput('');
+              };
+
+              // function filters items off arr which chars are not equal to filteredInput
+              const filterArr = arr => {
+                return arr.filter(
+                  item =>
+                    item.id.slice(0, filteredInput.length) === filteredInput,
+                );
+              };
```

Comment:
This is such a great idea! Nicely done! 👏

Reply:
I like your nifty solution here! 🎉

Reply:
Nice nice nice :)